src/Compositor.cpp

```diff
@@ -280,6 +280,9 @@ void CCompositor::startCompositor() {
 
     wlr_xcursor_manager_set_cursor_image(m_sWLRXCursorMgr, "left_ptr", m_sWLRCursor);
 
+    // TODO: Is this the ideal place to put this?
+    m_bAutoBackAndForth = g_pConfigManager->getConfigValuePtr("general:auto_back_and_forth")->intValue;
```

Comment:
wtf

Reply:
Where can I read this in then? I haven't been able to determine how configuration parameters are read in and initialised.